server/__tests__/deepseek.test.ts

```
@@ -22,3 +36,37 @@
   expect(spy).toHaveBeenCalled();
   spy.mockRestore();
 });
+
+test('deepseek throws when API key missing', async () => {
+  const original = process.env.DEEPSEEK_API_KEY;
+  try {
+    process.env.DEEPSEEK_API_KEY = undefined as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)